prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -126,9 +126,12 @@ class FHIRConverter(
             // TODO: https://github.com/CDCgov/prime-reportstream/issues/14287
             FhirPathUtils
 
-            val fhirBundles = process(format, queueMessage, actionLogger)
+            val processedItems = process(format, queueMessage, actionLogger)
 
-            if (fhirBundles.isNotEmpty()) {
+            // processedItems can be empty in two scenarios:
+            // - the blob had no contents, i.e. an empty file was submitted
+            // - the format is HL7 and the contents were not parseable, so the number of items is unknown
```

Comment:
Third scenario: We received an unsupported format, like CSV

Reply:
Good call... I'll add a comment